runtime/src/bank.rs

```diff
@@ -3257,6 +3281,39 @@ impl Bank {
         self.set_compute_budget(compute_budget);
     }
 
+    fn reconfigure_token2_native_mint(self: &mut Bank) {
+        let reconfigure_token2_native_mint = match self.operating_mode() {
+            OperatingMode::Development => true,
+            OperatingMode::Preview => self.epoch() == 93,
+            OperatingMode::Stable => self.epoch() == 74,
```

Comment:
Epoch 74 may be too tight for mainnet-beta.  ~15 hours away

Reply:
I'll check in the morning but I think we're "lucky" and blocks are being produced much slower right now, so we may have more than ~15 hours.

Reply:
Nope, my last comment was incorrect.  Epoch 74 is *way* to close now, this'll have to wait until Epoch 75 to activate